src/Poster/Poster.test.js

```diff
@@ -23,14 +23,66 @@ describe.only('Poster Component', () => {
         <Poster poster={poster} key={poster.id} displayMovieInfoPage={jest.fn()} />
       </Router>
     )
-      
+
     const rating = screen.getByText('9.0')
     const image = screen.getByAltText('Greenland')
 
     expect(rating).toBeInTheDocument()
     expect(image).toBeInTheDocument()
   });
 
+  it('should have the correct content when rendered when userFavorites are available', () => {
+    const ratingMatch = { movie_id: 149, rating: 10 }
+    const isFavorite = true
+    render(
+      <Router>
+        <Poster
+        poster={poster}
+        key={poster.id}
+        userRating={ ratingMatch }
+        setMovieView={ jest.fn() }
+        setFavoritesView={ jest.fn() }
+        isCurrentUser={ jest.fn() }
+        isFavorite={ isFavorite }
+        toggleUserFavorite={ jest.fn() }
+        filterFavorites={ jest.fn() } />
```

Comment:
Didn't realize you could pass in the props like this! That's really cool, smart use of this.